CHANGELOG.md

```diff
@@ -43,6 +44,8 @@ Here is a template for new release sections
 - `LIST_OF_NEW_PARAMETERS` renamed `EXTRA_CSV_PARAMETERS` and moved from `A1` to `constants.py
 ` (#384)
 - Order of parameters in tests/inputs, fixed missing parameters  (#384)
+- Only a single output flow (instead of multiple possible) as discussed in #149  (#391)
```

Comment:
Is the changes in the PR only regarding sources, or applicable for all assets?

I was wondering if there might be an instance that a dispatchable source needs two outputs. I came up with this: A fuel source is needed to supply one DG (one bus) and then a BHKW, which also can be powered with diesel fuel (ie. a seperate bus). I think it can happen, and with this edits we will have to keep in mind that we will need to fuel surces, ie. one for the DG and one for the BHKW. Fine for now, but if the fuel source is a fuel tank with this and that capacity, then it will be difficult to put a constraint on the annual fuel consumption/needed joint capacity.

Reply:
> Is the changes in the PR only regarding sources, or applicable for all assets?

This refers to sources. I updated the changelog in `dev`, thanks for noticing!